.clang-format

```diff
@@ -0,0 +1,67 @@
+BasedOnStyle: Google
+AccessModifierOffset: -1
+AlignAfterOpenBracket: DontAlign
+AlignConsecutiveAssignments: None
+AlignOperands: DontAlign
+AllowAllArgumentsOnNextLine: false
+AllowAllConstructorInitializersOnNextLine: false
```

Comment:
I'd not change constructor initializer lists, because different ones look good differently. Maybe removing this option has this effect?
```suggestion
```

Reply:
👍  [review changes](https://github.com/apache/nifi-minifi-cpp/pull/1876/commits/3884817a4647c54423275e2aded95b153e5dda8c)